langchain/src/document_loaders/fs/unstructured.ts

```diff
@@ -15,11 +15,16 @@ interface Element {
 }
 
 export class UnstructuredLoader extends BaseDocumentLoader {
-  constructor(public webPath: string, public filePath: string) {
+  constructor(
+    public webPath: string,
+    public filePath: string,
+    public apiKey: string = ""
```

Comment:
Can we put this in an options object arg?

Reply:
Sure thing!

Reply:
That’s added to an options arg now

Reply:
@nfcampos - Checking in for re-review on the options kwarg whenever you have bandwidth.